map event listeners enabled

pavvell opened the pull request:
Yandex maps service emits events that you may want to listen in you controllers/directives (like when user clicks on a balloon for example). This functionality is enabled now, you can listen any event you want by adding a list of them in new "events" attribute. [Event Docs](https://tech.yandex.ru/maps/doc/jsapi/2.0/ref/reference/Event-docpage/).

just-boris commented:
Thank you for a pull-request! The idea looks good, but it needs better implementation.

First of all, how are you going to detect map instance if you have more than one map on some page?

I think it would be better to do it by attributes:

```html
<yandex-map ymap-balloonopen="doSomething($event)"></yandex-map>
```

It is very similar how input directive in core angular works.

```
<input type="text" ng=model="myModel" ng-change="doSomething($event)">
```

You can see how it is implemented in [angular sources](https://github.com/angular/angular.js/blob/master/src%2Fng%2Fdirective%2FngEventDirs.js#L49) and do it here in the similar fashion

pavvell commented:
Good point! Working on it, will publish soon.

pavvell commented:
Hey, Boris! I've updated the event listeners mechanism, now it works as you recommended:

```html
<yandex-map ymap-balloonopen="doSomething($event)"></yandex-map>
```
To achieve this functionalty i've added new constant "EVENTS" and configurable parameter "eventPrefixInDirective" in ymapsConfig constant.

I've added description in readme as well.

just-boris commented on angular-ymaps.js:
```diff
@@ -73,7 +73,15 @@ angular.module('ymaps', [])
     fitMarkersZoomMargin: 40,
-    clusterize: false
+    clusterize: false,
+		eventPrefixInDirective: 'ymap'
```
There are tab symbols. Should be four spaces.

just-boris commented:
Thank you, good job!

But `registerEventEmitters` looks like a hack. You can do everything just in the controller, if you need attributes, you can inject them in the controller via `$attrs` injectable. 

Also, you should write few unit tests that `ymap-balloonopen` works, but something like `ng-balloonopen` doesn't.

And feel free to ask me some questions if you need.

pavvell commented:
Got it! Thanks for the reasonable feedback. Will update soon.
